package.json

```diff
@@ -6,11 +6,18 @@
   "author": "Shahed Nasser",
   "license": "ISC",
   "scripts": {
+    "build": "parcel build index.html",
+    "dev": "parcel index.html --open && lite-server",
+    "start": "npm run build && npm run dev && npm run watch:all",
```

Comment:
the "npm run watch:all" will never work, as the execution is stuck in "npm run dev" so maybe you can use "parallelshell" here for "npm run dev" and "npm run watch"